src/resqml2_0_1/EarthModelInterpretation.h

```diff
@@ -73,6 +73,14 @@ namespace RESQML2_0_1_NS
 
 		void pushBackStratiOccurence(class StratigraphicOccurrenceInterpretation * stratiOccurence);
 
+		// Rock Fluid
+		void setRockFluidOrganizationInterpretation(class RockFluidOrganizationInterpretation* rockFluid);
+
+		bool hasRockFluid() const;
```

Comment:
Naming should be consistent.
hasRockFluid -> hasRockFluidOrganizationInterpretation
getRockFluid -> getRockFluidOrganizationInterpretation